docs/source/index.rst

```diff
@@ -1,3 +1,14 @@
+=====================
```

Comment:
Also, for the same reasons, if we find the time to do it, we should try to get all the release notes and archive it in the docs.  Other than that, it looks great.